app/js/actions/init.js

```diff
@@ -27,10 +27,16 @@ var _actions = keymirror({
     pauseContainer: null,
     unpauseContainer: null,
 
-    listImages: null
+    listImages: null,
+
+    // Beacons
+    createBeacon: null,
```

Comment:
I was just thinking the other day we should probably name these like [noun][verb]. (i.e. `beaconCreate`) Once this list gets larger, I think it'll be easier to read.